toolkit/sharedlib/src/main/java/com/arcgismaps/toolkit/ui/expandablecard/ExpandableCard.kt

```diff
@@ -98,12 +101,12 @@ private fun ExpandableHeader(
     title: String = "",
     description: String = "",
     expandable: Boolean,
-    colors: ExpandableCardColors,
+    colors: ExpandableCardColorScheme,
```

Comment:
```suggestion
    colors: ExpandableCardColorScheme,
    typography: ExpandableCardTypography
```
This would need to also accept the typography.